.github/workflows/build.yml

```diff
@@ -27,30 +27,59 @@ jobs:
         variant: [aws-k8s-1.18, aws-k8s-1.19, aws-k8s-1.20, aws-k8s-1.21, aws-ecs-1]
         arch: [x86_64, aarch64]
         supported: [true]
+        fetch-upstream: ["false"]
```

Comment:
nit. our previous booleans (and values) were unquoted, but these ones are quoted. I think I would unquote these for consistency.

Reply:
The previous booleans values were used within the context of the github workflows, which are literal booleans. Since GH worklflows are defined in YAML, I can't use `false` and expect it to get translated to `"false"`, I could get a `1`, or another value that doesn't work for the environment variables passed to `cargo make`